src/components/Login.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useContext, useState } from "react";
+
+import ErrorMessage from "./ErrorMessage";
+import { UserContext } from "../context/UserContext";
+import { ApiClient } from "../api-client";
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const [, setToken] = useContext(UserContext);
+
+  const submitLogin = async () => {
+    const response = await ApiClient.submitLogin({
+      email: email,
+      password: password,
+    });
+    const data = await response.json();
+
+    if (!response.ok) {
+      setErrorMessage(data.detail);
+    } else {
+      setToken(data.access_token);
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    submitLogin();
+  };
+
+  return (
+    <div className="column">
+      <form className="box" onSubmit={handleSubmit}>
+        <h1 className="title has-text-centered">Login</h1>
+
+        <div className="field">
+          <label className="label">Email Address</label>
+          <div className="control">
+            <input
+              type="email"
+              placeholder="Enter email"
+              value={email}
+              onChange={(e) => setEmail(e.target.value)}
+              className="input"
+              required=""
+            />
+          </div>
+        </div>
+
```

Comment:
Lets move it to a separate component like `FormField` or something and there will be no need to repeat same component for Password.